source/AI.cpp

```diff
@@ -4622,3 +4637,51 @@ void AI::UpdateOrders(const Ship &ship)
 		order.targetSystem = ship.GetSystem();
 	}
 }
+
+// Job / Mission NPC blocks may use keywords (waypoint, visit, land) to define travel plans.
+void AI::IssueNPCOrders(Ship &ship, const System *waypoint, const std::map<const Planet *, bool> stopovers)
```

Comment:
I think I'd rename waypoint here, maybe to destination?
bc we dont know if its a waypoint or just the destination, right?

Reply:
At the same time, a destination is a waypoint, isn't it?

Maybe something like `targetSystem` would make more sense? Although it's used just above...

Reply:
Targetsystem's good